docs/development/repository_structure.md

```diff
@@ -0,0 +1,92 @@
+# Repository Structure
+
+The repository for Tractus-X EDC can be found [here](https://github.com/eclipse-tractusx/tractusx-edc).
+It contains the following components:
+
+## EDC Extensions
+
+The core EDC is extensible by design.
+Tractus-X EDC provides such extensions.
+These extensions and their documentation are available
+[here](https://github.com/eclipse-tractusx/tractusx-edc/blob/main/edc-extensions/README.md).
+
+
+### Business Partner Validation
```

Comment:
Keep this detail about extensions will make it outdated by definition, my suggestion would be eventually put these into a `README.md` into the `edc-extensions` folder (see previous comment)